src/pudl/etl.py

```diff
@@ -155,25 +135,25 @@ def _etl_eia(
 ###############################################################################
 
 
-def _read_static_tables_ferc1() -> dict[str, pd.DataFrame]:
-    """Populate static PUDL tables with constants for use as foreign keys.
+def _read_static_tables_ferc1():
+    """Compile static tables for FERC1 for foriegn key constaints.
 
-    There are many values specified within the data that are essentially
-    constant, but which we need to store for data validation purposes, for use
-    as foreign keys.  E.g. the list of valid EIA fuel type codes, or the
-    possible state and country codes indicating a coal delivery's location of
-    origin. For now these values are primarily stored in a large collection of
-    lists, dictionaries, and dataframes which are specified in the
-    pudl.metadata module.  This function uses those data structures to
-    populate a bunch of small infrastructural tables within the PUDL DB.
+    This function grabs static encoded tables via :func:`_read_static_encoded_tables`
+    as well as two static tables that are non-encoded tables (``ferc_accounts`` and
+    ``ferc_depreciation_lines``).
     """
-    return {
-        "ferc_accounts": FERC_ACCOUNTS[["ferc_account_id", "ferc_account_description"]],
-        "ferc_depreciation_lines": FERC_DEPRECIATION_LINES[
-            ["line_id", "ferc_account_description"]
-        ],
-        "power_purchase_types_ferc1": CODE_METADATA["power_purchase_types_ferc1"]["df"],
-    }
+    static_table_dict = _read_static_encoded_tables("static_ferc1")
+    static_table_dict.update(
+        {
+            "ferc_accounts": FERC_ACCOUNTS[
+                ["ferc_account_id", "ferc_account_description"]
+            ],
+            "ferc_depreciation_lines": FERC_DEPRECIATION_LINES[
+                ["line_id", "ferc_account_description"]
+            ],
+        }
+    )
```

Comment:
If we created another constant that just included the selected columns here, could we avoid the need for data source specific functions / special casing here?

Reply:
Ah, no. The problem is that the different types of static tables are (also) stored / accessed in different ways.